### ISSUE TITLE
[Bug][A11Y] Keyboard focus and VoiceOver focus issue on chip component

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-ios#914]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14+

### OS Version

iOS18

### App version

0.18.0

### App build number

1755794658

### App build type

beta

### Describe the issue

Using a keyboard (Full Keyboard Access activated), when I interact with a chip component in the settings configuration, just after the chip selection, the focus is not set on the entire chip component but only on the text inside, I need to wait for 2 seconds or press once the arrow key to retrieve the focus on the entire component. 

Same thing with Voice Over, when activating the chip, the focus is set around text only and gets correct after 2 seconds. But if I move the focus before that, the focus is moved either to the color palette component or to the tokens menu button.

Example : 

**Incorrect keyboard focus just after chip activation**
<img width="120" height="50" alt="" src="https://github.com/user-attachments/assets/c3ff1eca-160d-4262-993b-16246d0c3441" />

**Correct keyboard focus after moving with arrow key**
<img width="113" height="44" alt="" src="https://github.com/user-attachments/assets/41d09129-3810-4629-ac95-362608c8e365" />

### Expected Behavior

after activation, the focus should be put on the entire component, not only the text

### Accessibility settings

Full keyboard access

**Comment by pya35** [comment]
linked to #407
